packages/graph/hash_graph/lib/graph/src/shared/identifier.rs

```diff
@@ -0,0 +1,57 @@
+use serde::{Deserialize, Serialize, Serializer};
+use serde_json;
+use type_system::uri::VersionedUri;
+use utoipa::{openapi, ToSchema};
+
+use crate::knowledge::EntityId;
+
+// TODO - This is temporary and introduced for consistency, we need to introduce actual IDs for
+//  links, should be revisited as part of https://app.asana.com/0/0/1203157172269853/f
+#[derive(Debug, Clone, PartialEq, Eq, Hash, Serialize, Deserialize, ToSchema)]
```

Comment:
I've looked a little at the `LinkId` and its usage in `GraphElementIdentifier`. 
I think we can get rid of `ToSchema` here and the `schema` attribute below? `GraphElementIdentifier` is implementing `ToSchema` as `SchemaType::String` anyway :)

Reply:
It's possible you're right, but this is only for a temporary struct and it will be removed as one of our next ups so I'd suggest we don't waste time on it